templates/base.html

```diff
@@ -47,7 +47,7 @@
     integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
     crossorigin="anonymous">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <link href="/static/styles.css" rel="stylesheet">
+  <link href="/static/styles.css?1" rel="stylesheet">
```

Comment:
this (and the similar line on the js file) is obviously not the right way to do things. do we need to add some kind of cache-control to our static directory? not totally sure how to do that... i did need to use shift+reload while i was debugging in order to get my code to work.

Reply:
You can disable the cache when the dev tools are open -- there's a checkbox under "Network" in the "Preferences" page of the "Settings" UI in the dev tools (hit the gear icon):

<img width="1114" alt="Screen Shot 2020-12-22 at 11 40 41 AM" src="https://user-images.githubusercontent.com/207390/102911840-9d2a5d00-444a-11eb-9d09-ad8ad2e55b94.png">

Reply:
Yeah I was not concerned about my dev environment. Rather I was concerned about deploying new code and having a viewer display it with cached static files.

it's actually even worse than i thought b/c of the cloudflare caching - if we deploy new static files without modifying the cachebreaker here, then everyone (even people who never previously saw the site) will continue to get served the old version of the static files until cloudflare invalidates its cache.

given we've just got two weeks til the election, i'm not sure it's totally necessary to fix this, but i think in an ideal world we'd probably have some kind of build process that calculates a hash of the static file contents and appends it to the url here?

Reply:
The cloudflare cache is 30 minutes and the [in-browser cache is 15 minutes](https://github.com/vote/gaballot/blob/main/app.py#L70) so i'm not too worried. We can also manually invalidate the cloudflare cache from the cloudflare console. If we want to, we could look into something like this to automatically invalidate the cache on deploy: https://elements.heroku.com/buildpacks/jamesmartin/cloudflare-cache-purge-buildpack